spyne/decorator.py

```diff
@@ -354,6 +363,8 @@ def rpc(*params, **kparams):
         to override it for ``@rpc`` methods. It could be necessary to override
         it for ``@mrpc`` methods to add events and other goodies.
     :param _service: Same as ``_service``.
+    :param _message_part_name: Overrides the part name attribute of input/output
```

Comment:
this should be renamed to `_wsdl_part_name` and the docstring should make it clear that this is a WSDL-only change.